livre/eq/zeq/communicator.cpp

```diff
@@ -259,7 +259,7 @@ class Communicator::Impl
         // Receiving all queued events from all receivers without blocking.
         for( auto subscriber : subscribers )
             while( subscriber->receive( 0 ))
-                /*nop*/ ;
+                _config.sendEvent( REDRAW );
```

Comment:
So what do you suggest? We can't possibly do a full manual test for each seemingly innocent feature.